tests/sphere_lithostatic_pressure_2d.prm

```diff
@@ -0,0 +1,152 @@
+# this is a version of the stokes' law input file
+# but with less mesh refinement
```

Comment:
Could you also update the comments in the test input files, just with a very short description what the test is for?